src/Analyzers/CSharp/Analyzers/CSharpAnalyzers.projitems

```diff
@@ -95,6 +95,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)UseIndexOrRangeOperator\Helpers.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)UseIndexOrRangeOperator\MemberInfo.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)UseInferredMemberName\CSharpUseInferredMemberNameDiagnosticAnalyzer.cs" />
+    <Compile Include="..\..\..\Analyzers\CSharp\Analyzers\UseIsNullCheck\CSharpUseIsNullCheckOverIsObject.cs" />
```

Comment:
path is odd here.

Reply:
This was auto-generated by Visual Studio. Will fix.